### DISCUSSION TITLE
Github image camo as unblocker

### DISCUSSION THREAD

**Discussion by Inglan2** [3kh0/ext-remover#1105] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
if i add an image from an external url on github and it is on a blocked url, it still shows. i can see its proxying the image with `camo.githubusercontent.com`.

maybe we could use this to unblock something.

it looks like the url is encoded or has a one time code, for example
```
https://camo.githubusercontent.com/1fe49c25d7611bbd8f10fe7c9b12112abc4268e1dc4dc849a1484cf97fe2d409/68747470733a2f2f73747265616b2d73746174732e64656d6f6c61622e636f6d2f3f757365723d44656e766572436f64657231267468656d653d6461726b
```
so it might not be possible.

but it does look like the url stays the same so maybe it is encoded

**Comment by s0urce-c0de** [comment]
Hex? Hash? Also just use `data:text/html;,<img src="imglink">`.

**Reply by Inglan2** [reply]
Prob not a hash

**Reply by s0urce-c0de** [reply]
probably, but maybe we have to make make a request to the server to register hash? also look at this URL: `https://classroom.google.com/webthumbnail?url=https://my.url/path`
